app/uk/gov/hmrc/digitalservicestax/views/uniform/string.scala.html

```diff
@@ -35,6 +35,20 @@
 
 @telInput=@{key.contains("phoneNumber")}
 
+@autoComplete=@{key match {
+    case _ if key.contains("forename") => "given-name"
```

Comment:
I know there' not much difference in it, but I'f prefer if these lines read like:
   `case k if k.contains("forename") => "given-name"`